Library/Formula/git.rb

```diff
@@ -1,7 +1,7 @@
 class Git < Formula
   homepage "http://git-scm.com"
```

Comment:
@DomT4 seems like that should be done as a separate commit/change/PR from just bumping the versions doesn't it? 

That being said, I'm happy to tack on the change to this